src/platform.ts

```diff
@@ -0,0 +1,20 @@
+import { createHash } from 'crypto';
```

Comment:
This generates to https://gist.github.com/joscha/b2f96866e4fb29c9dfadec7472ef0eb6 - you can see how close it is to the original `lib/platform.js`